cea/optimization/slave/cooling_resource_activation.py

```diff
@@ -302,7 +297,7 @@ def cooling_resource_activator(Q_thermal_req,
                                                    T_district_cooling_supply_K,
                                                    VCC_T_COOL_IN,
                                                    size_AS_BaseVCC_W,
-                                                   VCC_chiller)
+                                                   VC_chiller)
```

Comment:
Should we maybe have a consistent convention? "VCC_chiller" is redundant, but is having some variables called "VC_chiller" and some "VCC_whatever" necessary? Maybe we can just rename "VCC_chiller" to "VCC" instead?

Reply:
i will vote for "VC_chiller" since it kind of mirrors "absorption_chiller"

Reply:
Yes, my thinking was exactly what @shanshanhsieh pointed out.